test/distributed/_tensor/experimental/test_local_map.py

```diff
@@ -0,0 +1,200 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+# Owner(s): ["oncall: distributed"]
+
+import torch
+import torch.distributed._functional_collectives as funcol
+from torch.distributed._tensor import (
+    distribute_tensor,
+    init_device_mesh,
+    Replicate,
+    Shard,
+)
+from torch.distributed._tensor.debug import CommDebugMode
+from torch.distributed._tensor.experimental import local_map
+from torch.testing._internal.common_utils import run_tests
+from torch.testing._internal.distributed._tensor.common_dtensor import (
+    DTensorTestBase,
+    with_comms,
+)
+
+DEVICE_TYPE = "cpu"
```

Comment:
is there a particular reason to only test this in CPU? Please change this to align with other tests so that we can test in multi gpus

Reply:
the motivation of limiting it to CPU test is that I think the correctness `local_map` should be consistent on CPU and GPU therefore we only need to test it on CPU.